Test-Source/Remove-Registry.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+Import-Module -Force $PSScriptRoot/../Source/Docker.Build.psm1
```

Comment:
The doppelganger of `Remove-Prefix.Tests.ps1`. :sunglasses:

Reply:
I Will be removed in the other PR I have open which is in draft now

Reply:
And now I have also removed the file in this one :D Well spotted!